sdks/java/expansion-service/src/main/java/org/apache/beam/sdk/expansion/service/ExpansionService.java

```diff
@@ -371,6 +371,20 @@ default InputT createInput(Pipeline p, Map<String, PCollection<?>> inputs) {
       return extractOutputs(
           Pipeline.applyTransform(name, createInput(p, inputs), getTransform(spec)));
     }
+
+    default List<String> getDependencies(PipelineOptions options) {
```

Comment:
I don't think a transform author can override this. See here: https://docs.google.com/document/d/1F5CW2UxSE3Jwr3e9u5cFWHMqFVwBO_PgEYYG2CAtrlw/edit?disco=AAAAQhYlvZw